templates/static/product-detail.twig

```diff
@@ -83,7 +83,8 @@
         </article>
 
         {# Aside #}
-        <div class="entry-aside">
+        <div
+            class="entry-aside">
```

Comment:
Is line break here and spacing above intentional?

Reply:
Nope, this type of thing was happening when some of my VSCode extensions were in conflict. I've been slowly cleaning these up as I go along.